.github/workflows/ci-soteria.yml

```diff
@@ -5,14 +5,52 @@ on:
     branches: [master, v*.*]
   pull_request:
     branches: [master, v*.*]
-
+  
+env:
+  CARGO_TERM_COLOR: always
+  SOLANA_VERSION: "1.9.5"
+  
 jobs:
   build:
-    name: Soteria Vulnerability Scanner
+    name: Soteria
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-
-      - name: Soteria Scan
-        uses: silas-x/soteria-action@v0.2
-        # continue-on-error: true
+      - name: Check-out repo
+        uses: actions/checkout@v2
+      
+      - name: Cache Solana binaries
+        uses: actions/cache@v2
+        id: solana-cache
+        with:
+          path: ~/.cache/solana
```

Comment:
it sounds like below things install to `~/.local/share/solana` - maybe that needs to be added to the cache and then we could skip the on-cache-hit path? I'll check